src/contracts/FlowToken.cdc

```diff
@@ -169,38 +170,34 @@ pub contract FlowToken: FungibleToken {
     //
     // The numbers are arbitrary.
     //
-    init() {
-        // Initialize the totalSupply field to the initial balance
-        self.totalSupply = 1000.0
+    init(adminAcct: AuthAccount) {
+        self.totalSupply = 0.0
 
         // Create the Vault with the total supply of tokens and save it in storage
         //
         let vault <- create Vault(balance: self.totalSupply)
-        self.account.save(<-vault, to: /storage/flowTokenVault)
+        adminAcct.save(<-vault, to: /storage/flowTokenVault2)
```

Comment:
Using `flowTokenVault2` for now to avoid a conflict with the bootstrapped `flowTokenVault` resource that the emulator places in each account.